resources/dicts/patrols/general/border.json

```diff
@@ -1907,5 +1907,271 @@
                 "other_clan_rep": -1
             }
         ]
+    },
+    {
+        "patrol_id": "gen_bord_otherclanapp",
+        "biome": [ "any" ],
+        "season": [ "any" ],
+        "types": [ "border" ],
+        "tags": [ "other_clan" ],
+        "patrol_art": "bord_general_intro",
+        "min_cats": 3,
+        "max_cats": 6,
+        "min_max_status": {
+            "apprentice": [ 1, 5],
+            "warrior": [1, 5]
+        },
+        "weight": 20,
+        "chance_of_success": 40,
+        "intro_text": "The patrol reaches o_c's border just as a o_c patrol approaches from the other side. The cats of both Clans shift uneasily, waiting for someone to break the silence. app1 perks up when {PRONOUN/app1/subject} {VERB/app1/spot/spots} another apprentice in the o_c patrol.",
```

Comment:
patrols actually use "o_c_n" for the other clan name.  i know it's inconsistent with the moon events, I'm planning to fix that lmao